app/uk/gov/hmrc/fhregistrationfrontend/views/helpers/AddressInternational.scala.html

```diff
@@ -14,67 +14,95 @@
  * limitations under the License.
  *@
 
-@import uk.gov.hmrc.fhregistrationfrontend.views.helpers._
+@import uk.gov.hmrc.fhregistrationfrontend.views.html.helpers.CountryCodes
+@import uk.gov.hmrc.fhregistrationfrontend.views.helpers.CountryCodeParams
+@import uk.gov.hmrc.fhregistrationfrontend.views.ViewHelpers
+@import uk.gov.hmrc.govukfrontend.views.html.components._
 
-@(formDate: Form[_], params: uk.gov.hmrc.fhregistrationfrontend.views.helpers.AddressUKParams)(implicit message: Messages)
+@import uk.gov.hmrc.fhregistrationfrontend.forms.definitions.ContactPersonForm.otherInternationalContactAddressKey
+@(formDate: Form[_], params: uk.gov.hmrc.fhregistrationfrontend.views.helpers.AddressUKParams, viewHelpers: ViewHelpers)(implicit message: Messages)
 
 @fieldName = @{
-  params.context match {
-    case Some(context) => context + "_" + params.name
-    case _ => params.name
-  }
+    params.context match {
+        case Some(context) => context + "_" + params.name
+        case _ => params.name
+    }
 }
-<fieldset id="@{fieldName}-fieldset">
-<legend class="form-label-bold">@{params.label}</legend>
+<fieldset id="@{fieldName}-fieldset" class="govuk-fieldset">
+    <legend class="govuk-fieldset__legend govuk-fieldset__legend--s">@{params.label}</legend>
+    @viewHelpers.govukInput(Input(
+        id = s"$fieldName.Line1",
+        name = s"$fieldName.Line1",
+        label = Label(
+            isPageHeading = false,
+            classes = "govuk-!-font-weight-bold",
+            content = HtmlContent(s"${message("fh.address.line1.label")}")
```

Comment:
Can probably get rid of the HTMLContent now it is a plain message?

Reply:
sure 👍